src/java/voldemort/server/PosixHostMatcher.java

```diff
@@ -0,0 +1,19 @@
+package voldemort.server;
+
+import java.util.List;
+
+import voldemort.cluster.Node;
+
+
+public class PosixHostMatcher extends HostMatcher {
```

Comment:
Why does this class exist?  Under what conceivable circumstances would the hostname comparison mechanism be anything except the simple and obvious `node.getHost().equals(host)` you have here?  FQDNs are an *Internet* concept not a POSIX concept, they're not going to be different on Windows.  The `match()` method should be moved into the parent class which should be made concrete.